scripts/generateSupportedNetworks.js

```diff
@@ -0,0 +1,111 @@
+const shell = require('shelljs')
```

Comment:
Should this script (and other automation scripts) be within the .github folder instead?